### DISCUSSION TITLE
How to launch in CLI only

### DISCUSSION THREAD

**Discussion by Blebowski** [RTimothyEdwards/netgen#77] [General]
Hi, I am playing with netgen, and I am unable to find reference to its command line arguments?

How can I start it only in CLI? I tried with ssh session without graphics (no -Y nor -X switches), but
that throws an error.

**Comment by RTimothyEdwards** [comment]
netgen _only_ runs with a command line interface.  If you want it to come up without a Tk console terminal, then either do "netgen -noconsole" to run tclsh directly from the terminal, or else use "netgen -batch <command>" to run it in batch mode ("<command>" can be "source <filename>" to source an entire Tcl script).

**Comment by RTimothyEdwards** [comment]
For command-line arguments, go to http://opencircuitdesign.com/netgen and click on "Reference".

**Comment by Blebowski** [comment]
Hi, @RTimothyEdwards , thanks for reference.
